cli/flox-rust-sdk/src/models/environment/managed_environment.rs

```diff
@@ -822,8 +844,8 @@ impl ManagedEnvironment {
         // Fetch the remote branch into FETCH_HEAD
         self.floxmeta
             .git
-            .fetch_ref("origin", &format!("{sync_branch}:",))
-            .unwrap();
+            .fetch_ref("origin", &format!("+{sync_branch}:{sync_branch}",))
+            .map_err(ManagedEnvironmentError::FetchUpdates)?;
```

Comment:
blocking: a change like this needs to be mentioned in the commit message/PR description or in a separate commit